flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/dataset/DataSetCalc.scala

```diff
@@ -42,63 +41,40 @@ class DataSetCalc(
     traitSet: RelTraitSet,
     input: RelNode,
     rowRelDataType: RelDataType,
-    private[flink] val calcProgram: RexProgram, // for tests
+    calcProgram: RexProgram,
     ruleDescription: String)
-  extends SingleRel(cluster, traitSet, input)
+  extends Calc(cluster, traitSet, input, calcProgram)
```

Comment:
OK, sounds good to me. I was just curious, because I remembered we had some issues when `DataSetCalc` extended `Calc`. If everything works fine, I'm OK with this change.